static/js/layout.js

```diff
@@ -610,10 +610,10 @@ function addCommas(number) {
 
 function getIncomeChartValueString(value, currency, ethPrice) {
   if (this.currency === "ETH") {
-    return `${value.toFixed(5)} ETH`
+    return `${value.toFixed(5)} LYXt`
```

Comment:
Shouldn't LYXt be a variable so we can use it for mainnet and testnet?

Reply:
Yes, it could. That's how Bitfly should implement this explorer, to have an external config variable `tokenSymbol`, but now as you can see it's hardcoded.

Reply:
I'll merge this now, so we can deploy on `testnet`. And in next PR I can take care of tokenSymbol refactor (move to config).